src/content/pages/sprints.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: EuroPython 2024 Sprints
+subtitle: Organise or join a sprint in EuroPython 2024!
+social_card: sprints.png
+---
+
+import { Image } from "astro:assets";
+
+import sprintImage from "./images/sprint_collection_2420x1619.png";
```

Comment:
Is any photo from last year's sprints included in this collection photo?

Reply:
Pretty sure not since I made this photo years ago :smile:

Reply:
@hypha Opened a thread on Discord to make a new one!

Reply:
After your last comment only one file was left.